app/javascript/app/components/info-download-toolbox/info-download-toolbox-component.jsx

```diff
@@ -20,7 +24,7 @@ class InfoDownloadToolbox extends PureComponent {
           <Icon icon={iconInfo} />
         </Button>
         <Button
-          onClick={handleDownloadClick}
+          onClick={() => this.handleDownloadClick(slug)}
```

Comment:
I prefer to avoid declaring functions on render time because they will be treated as a different value so will trigger a child render every time too. So why not using `this.handleDownloadClick` and read the slug from the props in the function directly?

Reply:
👍